app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -165,10 +167,11 @@ def parse_env_variables(env_variables)
         end
       end
 
-      def parse_openshift_image(openshift_image)
+      def parse_openshift_image(openshift_image, options)
         id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name]
         ref = "#{ContainerImage::DOCKER_PULLABLE_PREFIX}#{id}"
-        new_result = parse_container_image(id, ref)
+        new_result = parse_container_image(id, ref, :store_new_images => options.store_unused_images)
```

Comment:
The argument name should match the config key, `:store_new_images` sounds to me like it should do something different than `store_unused_images`